metaci/repository/templates/repository/repo_plans.html

```diff
@@ -69,15 +76,15 @@
             <div class="slds-truncate"><span>{{ plan.context }}</span></div>
           </th>
           <th data-label="Active">
-            <div class="slds-truncate"><span>{{ plan.active }}</span></div>
+            <div class="slds-truncate"><span>{{ planrepo.active }}</span></div>
```

Comment:
What if the planrepo is active but the plan is not? Then it'll look active here even though the github hook will skip it.

Reply:
good catch.